src/MacroTools/Commands/Captions.cs

```diff
@@ -0,0 +1,38 @@
+﻿using MacroTools.CommandSystem;
+using MacroTools.Extensions;
+using static War3Api.Common;
+
+namespace MacroTools.Commands
+{
+  /// <summary>
+  /// A <see cref="CommandSystem.Command"/> that sets the player's camera to a specific height.
+  /// </summary>
+  public sealed class Captions : Command
+  {
+    /// <inheritdoc />
+    public override string CommandText => "captions";
+    
+    /// <inheritdoc />
+    public override bool Exact => false;
+  
+    /// <inheritdoc />
+    public override int MinimumParameterCount => 1;
+
+    /// <inheritdoc />
+    public override CommandType Type => CommandType.Normal;
+
+    /// <inheritdoc />
+    public override string Description => "Toggles the display of captions.";
```

Comment:
Let's keep the language really simple for people.
```suggestion
    public override string Description => "Turns dialogue captions on or off.";
```